Seravo/seravo-plugin
# Add widget class and some new dashboard widgets

#### What are the main changes in this PR?
Add Dashboard_Widgets class for wp-admin dashboard widgets class. Extend dashboard widgets class by adding PHP end of life version and low disk space notifications. Also, move PHP error count widget from login notifications to the dashboard widgets class together with the new site status widget. Includes translations. 

PHP EOL needs reworking when the common config file / internal implementation is done. 

##### Why are we doing this? Any context or related work?
See #517

#### Manual testing steps?
To test end of life, error count and low disk space widgets there are some requirements:
- Some PHP errors on log (artificial or not)
- Switch to PHP 7.2
- `const LOW_DISK_SPACE_USAGE = 0.9;` change the value to something like 0.1 for instance. See the line [here](https://github.com/Seravo/seravo-plugin/blob/9d37bbb5afa32bbb256733ed189e8e1635f54283/modules/dashboard-widgets.php#L24)

Note that disk space usage is stored as a transient for 15 minutes, thus you need to delete transients and purge cache if you want to re-calculate and usage results. 

#### Screenshots
PHP EOL and warnings + disk space widget. For site status widget see #508 
![Screenshot(3)](https://user-images.githubusercontent.com/16706187/114685315-6c807300-9d1a-11eb-9cf1-38a8c7bf3ae0.png)
